### PR TITLE
feat: Enable resuming ingestion of the EDD website

### PR DESCRIPTION
**Pull request by yoomlam**
## Ticket

https://navalabs.atlassian.net/browse/DST-564

## Changes

* Add `--resume` ingestion argument for EDD website ingestion, which takes a long time.
* Use `argparse` to parse ingestion arguments
* When `--resume` is used, commit the DB transaction for each Document and skip ingesting documents that already exist in the DB

## Testing

Start ingestion and wait until a few "Adding embeddings for ..." log messages have passed
```
make ingest-edd-web DATASET_ID="CA EDD test" BENEFIT_PROGRAM=employment BENEFIT_REGION=California FILEPATH=src/ingestion/edd_scrapings.json INGEST_ARGS="--resume"
```
Check the DB that entries have been added.
Re-run the ingestion using the same command. Expect to see `Skipping -- item already exists:` messages for documents that already exist.

Other ingestion pipelines still work.

### CODE REVIEW

File: app/tests/src/test_ingest_edd_web.py [hunk 1]
```diff
@@ -220,21 +220,15 @@ def test__ingest_edd(
         ]
 
 
-@pytest.mark.parametrize("file_location", ["local", "s3"])
-def test__ingest_edd_using_md_tree(
-    caplog, app_config, db_session, edd_web_local_file, edd_web_s3_file, file_location
-):
```
Comment (yoomlam) [inline app/tests/src/test_ingest_edd_web.py:226]: Don't need to test s3 file location since it's tested elsewhere

File: app/Makefile [hunk 2]
```diff
@@ -258,7 +258,7 @@ ingest-bem-pdfs: check-ingest-arguments
 	$(PY_RUN_CMD) ingest-bem-pdfs "$(DATASET_ID)" "$(BENEFIT_PROGRAM)" "$(BENEFIT_REGION)" "$(FILEPATH)"
 
 ingest-edd-web: check-ingest-arguments
-	$(PY_RUN_CMD) ingest-edd-web "$(DATASET_ID)" "$(BENEFIT_PROGRAM)" "$(BENEFIT_REGION)" "$(FILEPATH)"
+	$(PY_RUN_CMD) ingest-edd-web "$(DATASET_ID)" "$(BENEFIT_PROGRAM)" "$(BENEFIT_REGION)" "$(FILEPATH)" $(INGEST_ARGS)
```
Comment (ccheng26) [inline app/Makefile:261]: does `$(INGEST_ARGS)` also need to be in quotes?

Comment (yoomlam) [reply app/Makefile:261]: No, it is intentionally not in quotes in case there are multiple arguments.

**Review by KevinJBoyer — APPROVED** [review]
Good stuff, thank you!

**Pull request merged**